src/frontend/Komponenter/Personoversikt/Revurdering/LagRevurdering.tsx

```diff
@@ -46,17 +44,23 @@ interface IProps {
     valgtBehandlingstype: Behandlingstype;
     lagRevurdering: (revurderingInnhold: RevurderingInnhold) => void;
     settVisModal: (bool: boolean) => void;
+    behandlinger: Behandling[];
 }
 
 export const LagRevurdering: React.FunctionComponent<IProps> = ({
     fagsakId,
     valgtBehandlingstype,
     lagRevurdering,
     settVisModal,
+    behandlinger,
 }) => {
     const { toggles } = useToggles();
     const { axiosRequest } = useApp();
 
+    const harMigrering = behandlinger?.some(
+        (behandling) => behandling.behandlingsårsak === Behandlingsårsak.MIGRERING
```

Comment:
Hvordan er reglene her? 

Skal man ha mulighet å droppe å ta inn barn for alle behandlinger som er av typen migrering eller bare hvis man har migreringsbehandling _OG_ ingen barn fra før?

Tenker det er forskjell på om man har migrert en sak, revurdert med nye barn - så blir det en ny revurdering hvor det finnes et nytt barn igjen...